addons/awsebscsiprovisioner/awsebscsiprovisioner.yaml

```diff
@@ -7,8 +7,8 @@ metadata:
     kubeaddons.mesosphere.io/name: awsebscsiprovisioner
     kubeaddons.mesosphere.io/provides: storageclass
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.5.0-4"
-    appversion.kubeaddons.mesosphere.io/awsebscsiprovisioner: "0.5.0"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.7.0-1"
+    appversion.kubeaddons.mesosphere.io/awsebscsiprovisioner: "0.7.0"
     values.chart.helm.kubeaddons.mesosphere.io/awsebscsiprovisioner: "https://raw.githubusercontent.com/mesosphere/charts/957337e/stable/awsebscsiprovisioner/values.yaml"
```

Comment:
```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/awsebscsiprovisioner: "https://raw.githubusercontent.com/mesosphere/charts/ccf4f8a/stable/awsebscsiprovisioner/values.yaml"
```

Reply:
we should remove this ... from the addons :/.

Reply:
It's an annotation consumed by the UI. Or, at least, that was the intention. Is that still true? @mesosphere/sig-ksphere-ui

Reply:
So the UI needs the `default_values_file` metadata for specifically _catalog_ addons (so anything in kubeaddons-enterprise atm). I imagine this metadata is populated from this annotation by the catalog API? I do not see a reason we need to include them in kba though since those won't be deployed through the catalog?